configlet.nimble

```diff
@@ -0,0 +1,13 @@
+# Package
+version       = "0.1.0"
+author        = "ee7"
+description   = "A tool for managing Exercism language track repositories"
+license       = "AGPL-3.0-only"
+srcDir        = "src"
+
+namedBin["configlet"] = "configlet_v3"
```

Comment:
Good idea - I'll add a comment to clarify `namedBin`, since it's probably unfamiliar to most readers.

But when we rename, we'll also need to update the below files/lines. The one in the `.nimble` file is one of the easier ones to remember, since that's the one that controls the binary name.

```
.github/bin/create-artifact
 6:  BINARY_NAME="configlet_v3"

.gitignore
 5:  configlet_v3

README.md
 1:  # Configlet
 3:  This is a development version of Configlet for use with v3 of Exercism.
13:    configlet_v3 [global-options] <command> [command-options]
35:  To use the application in your track, you can copy the [`scripts/fetch-configlet_v3`](./scripts/fetch-configlet_v3) and/or [`scripts/fetch-configlet_v3.ps1`](./scripts/fetch-configlet_v3.ps1) files to your track's repository. Running either of these scripts will download the latest version of the `configlet_v3` tool to your track's `bin` directory.
39:  Bug reports and pull requests are welcome on GitHub at https://github.com/exercism/configlet-v3.

configlet.nimble
 8:  namedBin["configlet"] = "configlet_v3"

scripts/fetch-configlet_v3
 5:  readonly LATEST='https://api.github.com/repos/exercism/configlet-v3/releases/latest'
35:  FILENAME="configlet_v3-${OS}-${ARCH}.${EXT}"
47:          curl --header "$HEADER" -s --location "$URL" -o bin/latest-configlet_v3.zip
48:          unzip bin/latest-configlet_v3.zip -d bin/
49:          rm bin/latest-configlet_v3.zip

scripts/fetch-configlet_v3.ps1
 2:      $latestUrl = "https://api.github.com/repos/exercism/configlet-v3/releases/latest"
17:  $fileName = "configlet_v3-windows-$arch.zip"

tests/test_binary.nim
 7:    binaryName = "configlet_v3" & binaryExt
```

Reply:
Maybe add a todo to all those lines?

Reply:
I've added the comment about `namedBin`, but I lean against adding comments for the others.

That's because I don't want the list of todos to be the source of truth: when we do the rename later, we should grep for `configlet` rather than `todo` - it's easy to add more things that have to be changed at rename-time, and forget to add a todo.